tasks/package.yml

```diff
@@ -38,9 +38,8 @@
 - name: Install gunicorn
   pip:
     name: gunicorn
-    executable: pip3
-    extra_args: --user
     state: latest  # noqa package-latest
+    virtualenv: "{{ pretalx_virtualenv }}"
```

Comment:
You're using `|default(omit)` in some places but not in others, is that intentional?